deploy/kubedirector/kubedirector.hpe.com_kubedirectorclusters_crd.yaml

```diff
@@ -169,6 +169,19 @@ spec:
                       storageClassName:
                         type: string
                         minLength: 1
+                  blockStorage:
+                    type: object
+                    nullable: true
+                    required: [storageClassName, path]
+                    properties:
+                      path:
+                        type: string
```

Comment:
need regex/validation for path

Reply:
Hmm this should be changed when we add numDevices. What should reflect is just the prefix path, and we should incremental numbers. So use can say something like "/dev/kdb" and inside the container it should be /dev/kdb0, /dev/kdb1 etc

Reply:
`pathPrefix` now requires a minimum of one character. I think it is on the user to specify a valid path. Thoughts?

Reply:
what will be the status of cluster if invalid path is provided?

Reply:
Pod goes in Error state:

```
[root@mip-bd-vm226 kd-pvc]# kubectl get pod
NAME           READY   STATUS                       RESTARTS   AGE
kdss-2rglb-0   0/1     CreateContainerConfigError   0          52s
```


Describe yields:
```
  Warning  Failed                 6s (x7 over 66s)  kubelet, mip-bd-vm226.mip.storage.hpecorp.net  Error: error DevicePath `invalid0` must be an absolute path
```

Reply:
That makes sense. First character has to be "/"

We can mandate this to be /dev/kdb or something like that also

Reply:
Okay. I shall add a regex check to ensure that the first character is `/`
Sounds good?

Reply:
I think that's fine for now.